cmd/expired-authz-purger/main.go

```diff
@@ -26,10 +26,12 @@ type eapConfig struct {
 
 		Syslog cmd.SyslogConfig
 
-		GracePeriod cmd.ConfigDuration
-		BatchSize   int
-		MaxAuthzs   int
-		Parallelism uint
+		GracePeriod           cmd.ConfigDuration
+		BatchSize             int
+		MaxAuthzs             int
+		Parallelism           uint
+		PendingCheckpointFile string
+		FinalCheckpointFile   string
```

Comment:
Is there an example config file that needs to be updated with `PendingCheckpointFile` and `FinalCheckpointFile` examples?